extensions/oidc/runtime/src/main/java/io/quarkus/oidc/runtime/TenantConfigBean.java

```diff
@@ -32,6 +38,7 @@ public final class TenantConfigBean {
                 oidc.getDefaultTenantConfig());
         this.defaultTenant = tenantContextFactory.createDefaultTenantConfig(oidc.getStaticTenantConfigs(),
                 oidc.getDefaultTenantConfig());
+        this.newBackChannelLogoutPathEvent = Arc.container().beanManager().getEvent().select(NewBackChannelLogoutPath.class);
```

Comment:
I wonder if we can make it similar to how it is done in `LazyTenantConfigContext`, where it is acquired in scope of the code which needs it and then we can push it to the utility class, like `LogoutUtils` I'm adding in the RP initiated form post logout PR ? 
It will likely be nearly as effective. Given that a back channel logout would only be used in rather advanced applications only, most tenants don't even need this event selection too...
Does it sound reasonable ?

Reply:
Yeah, it does. I think it is perfectly fine to do that as in `LazyTenantConfigContext` because as you say, usually it is not used.